### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by twodarek**
It works for Mac now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ Every time you open a new terminal window, you will have to activate the virtual
 
     source venv/Scripts/activate
 
+If you are on Mac OSX, you will need to activate the virtual environment with this instead:
```
Comment (jglamine) [inline README.md]: Also mention Linux, cause it's the same.

**Comment by jglamine** [comment]
Looks good. Squash your two commits and see my comment about Linux.

**Comment by twodarek** [comment]
PTAL

**Comment by twodarek** [comment]
PTAL

**Comment by twodarek** [comment]
PTAL

**Comment by twodarek** [comment]
"If you are not on Windows..."

**Pull request merged**